packages/wmr/src/wmr-middleware.js

```diff
@@ -490,7 +506,7 @@ export const TRANSFORMS = {
 						}
 					}
 
-					const modSpec = spec.startsWith('../') ? spec.replace(/..\/g/, '') : spec.replace('./', '');
```

Comment:
I have no clue how the `/..\/g/` regex ever made it into our code base. Its intention is pretty obvious, safe for fact that the regex syntax doesn't work that way in JavaScript :D

```txt
. matches any character (except for line terminators)
. matches any character (except for line terminators)
\/ matches the character / literally (case sensitive)
g matches the character g literally (case sensitive)
```

So this regex only matched literally strings like: `"fa/g"`

Reply:
heh - I should have caught that! looks like it snuck in in #552 🥸